Contributing/How-to-Get-Credit-for-Your-Contribution.md

```diff
@@ -1,16 +1,15 @@
-## Get Credit:  
-  
-GitHub automatically records all contributions to code and md-documents under the [Insights/Contributors](https://github.com/moja-global/About-moja-global/graphs/contributors) tab.  
+## Get Credit for your Contribution:
+
+GitHub automatically records all contributions to code and documentation under the [Insights/Contributors](https://github.com/moja-global/About-moja-global/graphs/contributors) tab.
 
 In addition, moja global is using the [All Contributors Bot](https://allcontributors.org/) to recognize all contributors for a wide range of [possible contributions](https://allcontributors.org/docs/en/emoji-key).  
 
-But if you would like to be recognized, please blow your own trumpet by follow the steps below:
+But if you would like to be recognized, please blow your own trumpet by following the steps below:
 
-1. After making your contribution (submitting your pull-request, replying to a question, resolving an issues, etc.) add the following sentence in a comment to an existing issue    
-`@all-contributors please add <@username> for <contributions>`
-1. `<@username>` can be any person or organisation with a GitHub account   
-1. `<contributions>` can be any word from [this list](https://allcontributors.org/docs/en/emoji-key) but the bot will also use [basic Natural Language Parsing](https://github.com/all-contributors/all-contributors-bot/blob/master/src/tasks/processIssueComment/utils/parse-comment/index.js) to determine your contribution intent.
-1. Submit your comment
-1. The `Allcontributors Bot` will submit a pull-request to include `<@username>` in the list of contributors 
-1. Once the pull-request is accepted by the maintainer of the repository, `<@username>` will be added to the list of contributors for the suggested `<contribution>`.
-  
+1.  After making your contribution (submitting your pull-request, replying to a question, resolving issues and more) add the following sentence in a comment to an existing issue:\
+    `@all-contributors please add <@username> for <contributions>`
```

Comment:
Did we have an issue open for Github Bots? I forget where.

Reply:
I will add it by today. Thanks for reminding me!